src/pages/workspace/workflows/approvals/WorkspaceWorkflowsApprovalsApproverPage.tsx

```diff
@@ -126,7 +125,13 @@ function WorkspaceWorkflowsApprovalsApproverPage({policy, personalDetails, isLoa
                         isSelected: selectedApproverEmail === email,
                         login: email,
                         icons: [{source: avatar ?? FallbackAvatar, type: CONST.ICON_TYPE_AVATAR, name: displayName, id: accountID}],
-                        rightElement: isAdmin ? <Badge text={translate('common.admin')} /> : undefined,
+                        rightElement: (
+                            <MemberRightIcon
+                                role={employee.role}
```

Comment:
I see this was `employee?.role` before - does it need to be that here as well?

Reply:
Probably just an oversight @rafecolton, we had a tight deadline back then 😅